Utilities/Analyzer.cc

```diff
@@ -9,28 +9,20 @@
 #include <iostream>
 #include <string>
 
-#include "DataFormat.cc"
+//#include "DataFormat.cc"
 #include "NanoAODReader.cc"
 // #include "BTag.cc"
 #include "Constants.cc"
 #include "ProgressBar.cc"
 // #include "ScaleFactor.cc"
 #include "DataSelection.cc"
-#include "PUReweight.cc"
```

Comment:
I've used the PU reweight branches Andrew added to the ntuples. Is there a need for that function or file?

Reply:
This is the module that reads from the PUReweights I made my own.
Since we are not sure what x-section Andrew's PU weight is corresponding to.

Reply:
I suggest you make a comparison of the contents in Andrew's branches to what you get from your module and present that in our next meeting, then.
I've used what's ready to use and in the skims right now, but you are right, that that doesn't prove it's correct.